connections.md

```diff
@@ -2,254 +2,7 @@
 
 A Connection is the practical application of a relationship in DID Communication. Having a connection means that each party in the relationship has a DID for the other parties, and parties can communicate securely using the keys and endpoints within each DID Document.
 
-In order to establish a new connection, we use the OutOfBand Protocol and the DID Exchange Protocol.
+In order to establish a new connection, Simply exchange a new message between parties. Knowing the DID of the other parties does not indicate any level of trust. Using the connection to establish a foundation of trust is the next step.
 
 
 
-#### OutOfBand Protocol
-
-Invitations are usually the first message passed between parties. They are passed out of band, such as in a QR code or a link that can either open in an app or in a browser containing getting started instructions.
-
-The OutOfBand Protocol exists as an independent protocol to allow for the invitation recipient to respond with any of the supported protocols.
-
-A Public DID functions as a standing invitation. The information within the resolved DID Document serves the same function as the information passed in OutOfBand protocol messages.
-
-TODO: Include details of the protocol.
-
-##### Ephemeral Mode
-
-Some interactions do not need an established connection to complete their purpose. These interactions occur in _ephemeral mode_ and use OutOfBand protocol messages for the bulk of the interaction.
-
-TODO: Include more details about ephemeral mode.
-
-#### DID Exchange Protocol
-
-##### 1. Exchange Request
-
-The exchange request message is used to communicate the DID document of the _invitee_ to the _inviter_ using the provisional service information present in the _invitation_ message.
```

Comment:
I assume this purpose of the DID Exchange Protocol is obsoleted by usage of initial-state.

I am wondering about the nuance of peer DIDs and initial-state after the connection is considered established. Should initial-state continue to be sent after that point?

Reply:
I think yes, though I think initial-state might not be the right term for that.

Reply:
If the parties have an implicit connection establishment model (on top of the 0RTT any message model), then perhaps they could choose to stop sending initial-state once established. Without knowledge that the other party has persisted then, yes, I guess they would have to include with every message.

Reply:
Agree. How the further changes to a did:peer did are communicated to the various parties is something that the method will need to define. Perhaps deltas a communicated in a query param as well, and dropped upon confirmation of receipt.